src/ErrorHandler.php

```diff
@@ -67,7 +67,7 @@ public function onShutdown()
         if ($error['type'] & error_reporting() === 0) {
          return;
         }
-        $exc = new Errors\Fatal($error['message'], $error['file'], $error['line']);
+        $exc = new Errors\Fatal($error['message'], $error['file'], $error['line'], $trace = array(), $error['type']);
```

Comment:
Thanks for doing this, man.

Could you also update `onError()` such that each of the Base objects that it creates is given the `$code` as the 5th argument to the constructor?